jerry-core/ecma/base/ecma-helpers.h

```diff
@@ -96,6 +96,16 @@ typedef enum
   ECMA_STRING_FLAG_MUST_BE_FREED = (1 << 3), /**< The returned buffer must be freed */
 } ecma_string_flag_t;
 
+/**
+ * Enumerable for string trimming location.
+ */
+typedef enum
+{
+  ECMA_TRIM_STRING_START = (1u << 0), /**< Trim only from the start */
+  ECMA_TRIM_STRING_END = (1u << 1), /**<  Trim only from the end */
+  ECMA_TRIM_STRING_STARTEND = (1u << 2), /**< Trim from both start and end */
+} ecma_string_trim_flags_t;
```

Comment:
No need for these, just call `ecma_string_trim_front`/`ecma_string_trim_back` where necessary, and change `ecma_string_trim_helper` to call both, and be always inline.